### DISCUSSION TITLE
Error when try to run rotor blade simulation PETSC ERROR: Caught signal number 7 BUS: Bus Error, possibly illegal memory access

### DISCUSSION THREAD

**Discussion by Ryupuriwatt** [mdolab/dafoam#605] [General]
I modify the tutorial of rotor blade and run it with my own mesh the error that I normally get is that "illegal memory access" and it stop running some time the error is occur at before primal solver running. But some time it occur at gradient calculation. After this error occur I check my computer and I see that memory has been use very high even I just open the normal google or after I close google memory that has been used still high. Could you please suggest me what is a possible solution to fix this problem?
![image](https://github.com/mdolab/dafoam/assets/115110434/8ffd1a49-9630-4dca-8935-328755b7f4d9)
![image](https://github.com/mdolab/dafoam/assets/115110434/df7f87d1-7572-4b3d-b669-5bdcc1af3e25)

**Comment by Ryupuriwatt** [comment]
Now I have add --memory=30g to the command to start using docker images but still get the following error sudo docker run -it --rm --memory=30g -u dafoamuser --mount "type=bind,src=$(pwd),target=/home/dafoamuser/mount" -w /home/dafoamuser/mount dafoam/opt-packages:v3.1.0 bash 
![image](https://github.com/mdolab/dafoam/assets/115110434/4b10ca41-264e-49b8-93fc-261d892ef030)

**Comment by ryupuriwat** [comment]
Profressor I try to run it with 3 different computers but still get the same problem so I really don't know where should I check the problem.

**Comment by friedenhe** [comment]
How many mesh cells do you have for this case? Please check: https://dafoam.github.io/mydoc_get_started_faq.html#how-many-cpu-cores-to-use-and-how-much-memory-does-it-need whether you have enough memory to run the case.

**Reply by ryupuriwat** [reply]
I have a virtual machine with VirtualBox that can utilize a maximum of 32 processors, while my physical computer has 64 CPU cores and 128 GB of RAM. Considering that I reserve one CPU core and allocate 2 GB of memory per 10,000 cells in my computational simulations, I estimate that I can effectively handle up to 320,000 cells in my virtual environment. Could you confirm if my estimation is correct?

**Reply by Ryupuriwatt** [reply]
I have another question. I have checked inside all folders but could not find where to modify the rotational speed of the rotor blade. Instead, what I found were parameters related to the rotational reference point, rotational direction, and inlet speed. Thank you for your assistance; I genuinely appreciate it.

**Reply by 15175829310** [reply]
Have you solved this problem? I also encountered this problem. I hope you can give me some advice.

**Comment by friedenhe** [comment]
I suggest you start with a smaller mesh and test. Also, you can change the rotation speed in MRFProperties.
